.github/workflows/complement.yml

```diff
@@ -0,0 +1,36 @@
+# This task does not run complement tests, see tests.yaml instead.
+# This task does not build docker images for synapse for use on docker hub, see docker.yaml instead
+
+name: Store complement synapse image in ghcr.io
+on:
+  push:
+    branches: ["master", "develop"]
```

Comment:
this is basically entirely up to you, but do you want to push images for tagged releases?
I could see it maybe being useful for comparing different releases if necessary.

Reply:
For my test cases, we probably only want to test current and next as well; if people want to investigate certain synapse versions then that can happen as a local build for now.